arduinolux/src/Debug/DebugTest.cpp

```diff
@@ -0,0 +1,14 @@
+#include"../StringConverter.h"
+#include <string>
+#include <iostream>
+
+int main() {
+	std::string testString = "Hello there world!";
+	do {
+		char* convertedString = StringConverter::ExtractCharArrayFromString(testString);
+
+		std::cout << convertedString << "\n";
```

Comment:
Just:
**inline char* ExtractCharArrayFromString(std::string testString){
     return testString.c_str();
}**
The output kept printing out looking like just a raw pointer (technically what it is, true, but not how std::cout should print a char*). Now I'm curious why that wasn't working though... may warrant some poking around after homework.